flutter_local_notifications/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [19.0.0-dev.4]
+
+* [Windows] Reworked the APIs around custom images and audio. Use the new `.asset()` or `.network()` constructors
```

Comment:
Looks like the PR introduces breaking changes between prerelease versions. Can you update the changelog entries to call these out more explicitly?